packages/routing-utils/src/schemas.ts

```diff
@@ -1,3 +1,99 @@
+export const mitigateSchema = {
+  description: 'Mitigation action to take on a route',
+  type: 'object',
+  additionalProperties: false,
+  required: ['action'],
+  properties: {
+    action: {
+      description: 'The mitigation action to take',
+      type: 'string',
+      enum: ['log', 'challenge', 'deny'],
+    },
+  },
+} as const;
+
+export const conditionValueSchema = {
```

Comment:
Let's be more specific here

```suggestion
export const hasValueSchema = {
```

Reply:
fixed.